src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataService.java

```diff
@@ -189,10 +189,14 @@ public String getCaseDataCaseRef(UUID caseUUID) {
     }
 
     public String getCaseDataField(UUID caseUUID, String key) {
-        log.debug("Looking up key {} for Case: {}", key, caseUUID);
-        Map<String, String> dataMap = getCaseData(caseUUID).getDataMap(objectMapper);
+        return getCaseDataField(getCaseData(caseUUID), key);
+    }
+
+    public String getCaseDataField(CaseData caseData, String key) {
```

Comment:
Is this abstraction actually necessary, Line 308 does the same `getCaseData` call. Worth just moving this within the UUID for now. We can always add this back in the future...